tests/Integration/TestApp/Container/Compiler/CompilerPass.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Mleko\Narrator\Bundle\Tests\Integration\TestApp\Container\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class CompilerPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        foreach ($container->getDefinitions() as $definition) {
+            $definition->setPublic(true);
```

Comment:
This will make all services public.
But good point that https://symfony.com/blog/new-in-symfony-3-4-services-are-private-by-default
I will update bundle